wireguard/build-wireguard-go.sh

```diff
@@ -2,39 +2,57 @@
 
 # This script is used to build wireguard-go libraries for all the platforms.
 
+TEMP=$(getopt -o aiz --long android,docker,ios,amnezia \
+              -n 'build-wireguard-go.sh' -- "$@")
+
+if [ $? != 0 ] ; then echo "Terminating..." >&2 ; exit 1 ; fi
+
+# Note the quotes around '$TEMP': they are essential!
+eval set -- "$TEMP"
+
+ANDROID_BUILD=false
+IOS_BUILD=false
+DOCKER_BUILD=true
+AMNEZIA_BUILD=false
+while true; do
```

Comment:
Been meaning to do that. We should extract arts parsing into main :)

Reply:
I can split this out into it's own little PR

Reply:
That would be great!

Reply:
PR created here - https://github.com/nymtech/nym-vpn-client/pull/1432

I think it now works for the enabled build platforms